speed-dreams/source-2.2.3/data/data/menu/ResearcherMenu.xml

```diff
@@ -6,31 +6,27 @@
 
     <section name="dynamic controls">
 
-        <!-- Checkboxes for selecting a Task -->
-        <section name="CheckboxTaskLaneKeeping">
-            <attstr name="type" val="check box"/>
-            <attstr name="text" val="Lane keeping"/>
-            <attstr name="tip" val="Select the Lane Keeping task"/>
+        <!-- RadioButtonList for selecting a Task -->
+        <section name="TaskRadioButtonList">
+            <attstr name="type" val="radio button list"/>
+            <section name="sub texts">
+                <attstr name="text0" val="Lane Keeping"/>
+                <attstr name="text1" val="Speed Control"/>
+            </section>
+            <section name="sub tips">
+                <attstr name="tip0" val="Select the Lane Keeping task"/>
+                <attstr name="tip1" val="Select the Speed Control task"/>
+            </section>
```

Comment:
Something like this would make more sense, and you can remove the 'amount' parameter in that case.
Now the different options with their attributes can be defined in a single place.
```suggestion
            <section name="options">
                <section name="0">
                   <attstr name="text" val="Lane Keeping"/>
                   <attstr name="tip" val="Select the Lane Keeping task"/>
                </section>
                <section name="1">
                   <attstr name="text" val="Speed Control"/>
                   <attstr name="tip" val="Select the Speed Control task"/>
                </section>
            </section>
```